dotcom-rendering/src/components/EuropeLandingModal.importable.tsx

```diff
@@ -323,6 +360,11 @@ export const EuropeLandingModal = ({ edition }: Props) => {
 						/>
 					</RadioGroup>
 					<Button
+						data-link-name={nestedOphanComponents(
+							'eu-modal',
+							'switch-edition',
+							'confirm-' + selectedEdition,
```

Comment:
This will be confirm-INT/confirm-EUR/confirm-US/confirm-UK/confirm-AU

Reply:
Hi Oliver, I don't have a enough context for this change. Are there any specific concerns you have?

Reply:
Good question! Looking at the code it looks like this will be evaluated fresh each click event and therefore should be fine, but I don't have a good way to test/verify this. Perhaps you can try this locally and check the outgoing Ophan requests using the browser tools. Link to code: https://github.com/guardian/ophan/blob/9829623042b8beca483013fbd3053103ad1494e0/tracker-js/assets/coffee/ophan/click-path-capture.coffee#L33-L54

Reply:
Thanks! I'll give that a go!